plugins/WebsiteMeasurable/MeasurableSettings.php

```diff
@@ -235,7 +235,9 @@ private function makeExcludedUserAgents()
                 . '<br /><br />'
                 . Piwik::translate('SitesManager_GlobalListExcludedUserAgents_Desc')
                 . '<br />'
-                . Piwik::translate('SitesManager_GlobalExcludedUserAgentHelp2');
+                . Piwik::translate('SitesManager_GlobalExcludedUserAgentHelp2')
+                . Piwik::translate('SitesManager_GlobalExcludedUserAgentHelp3', "/bot|spider|crawl|scanner/i")
```

Comment:
Guess would make sense to insert a space between both strings.

```suggestion
                . Piwik::translate('SitesManager_GlobalExcludedUserAgentHelp2') . " "
                . Piwik::translate('SitesManager_GlobalExcludedUserAgentHelp3', "/bot|spider|crawl|scanner/i")
```

Reply:
Thank you for the review @sgiehl! I, too, am a bit busy at the moment but will be able to follow up this PR over the weekend.